spec/features/user_creates_account_spec.rb

```diff
@@ -0,0 +1,104 @@
+require 'rails_helper'
+
+feature 'user creates account' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.